eng/testing/performance/android_scenarios.proj

```diff
@@ -39,6 +39,12 @@
       <Command>$(Python) test.py devicestartup --device-type android --package-path pub\HelloAndroid.apk --package-name net.dot.HelloAndroid --scenario-name &quot;%(Identity)&quot; $(ScenarioArgs)</Command>
       <PostCommands>$(Python) post.py</PostCommands>
     </HelixWorkItem>
+    <HelixWorkItem Include="Build Time - Android HelloWorld">
+      <PayloadDirectory>$(WorkItemDirectory)</PayloadDirectory>
+      <PreCommands>cd $(ScenarioDirectory)helloandroid;mkdir traces;copy %HELIX_WORKITEM_ROOT%\androidHelloWorldBinlog\msbuild.binlog traces</PreCommands>
+      <Command>$(Python) test.py buildtime --scenario-name &quot;%(Identity)&quot; $(ScenarioArgs) --binlog-path msbuild.binlog</Command>
```

Comment:
Not sure I understand how this works, could you please clarify?
More specifically, why do we create `traces` folder and copy the binlog to it, when we later pass only `--binlog-path msbuild.binlog` to the `test` script, as if the binlog is in the current directory?
Couldn't we just pass `--binlog-path %HELIX_WORKITEM_ROOT%\androidHelloWorldBinlog\msbuild.binlog` ?

nit: Additionally, should we call the folder `binlogs` instead of `traces`?

Reply:
In `runner.py` we initialize the Startup tool which implements BuildTimeParser. Because we don’t actually run the app to measure the input for the tool, the script calls `parsetraces` which root directory is traces: https://github.com/dotnet/performance/blob/e19af5b0f10c7859915d2f02211c170f80b4c424/src/scenarios/shared/startup.py#L59-L63

Both the ios and android scenarios first create a traces directory and copy the .msbinlog file there. Later in the tool/script, that file is copied from traces into the current working directory.

Reply:
Thanks for the explanation!